src/webgpu/api/validation/capability_checks/features/texture_formats.spec.ts

```diff
@@ -171,6 +171,53 @@ g.test('canvas_configuration')
     }
   });
 
+g.test('canvas_configuration_view_formats')
+  .desc(
+    `
+  Test that configuring a canvas with view formats throws an exception if the required optional
+  feature is not enabled. Otherwise, a validation error should be generated instead of throwing an
+  exception.
+  `
+  )
+  .params(u =>
+    u
+      .combine('format', kOptionalTextureFormats)
```

Comment:
nit: Change this to `viewFormat` which would make the error below clearer.

Perhaps even change it to `viewFormats` (an array). This would allow adding some bonus test cases, e.g.:

```suggestion
      .combine('viewFormats', [
          ...kOptionalTextureFormats.map(fmt => [fmt]),
          ['bgra8unorm', 'bc1-rgba-unorm'],
          ['bc1-rgba-unorm', 'bgra8unorm'],
      ])
```